tests/base.py

```diff
@@ -27,7 +27,7 @@ class GoogleAdsBase(unittest.TestCase):
     INCREMENTAL = "INCREMENTAL"
     FULL_TABLE = "FULL_TABLE"
     START_DATE_FORMAT = "%Y-%m-%dT00:00:00Z"
-    REPLICATION_KEY_FORMAT = "%Y-%m-%dT00:00:00.000000Z"
```

Comment:
Let's discuss.  I assume this was the conversation with Dylan.

Reply:
This actually should get changed back according to that discussion.